fulp_modules/main_features/bloodsuckers/code/powers/feed.dm

```diff
@@ -332,14 +334,28 @@
 		if(ishuman(target))
 			target.adjustStaminaLoss(5, forced = TRUE) // Base Stamina Damage
 
-/datum/action/bloodsucker/feed/DeactivatePower(mob/living/user = owner, mob/living/target)
-	..() // activate = FALSE
+/datum/action/bloodsucker/feed/DeactivatePower(mob/living/user = owner)
+	var/mob/living/target = feed_target
+	. = ..() // activate = FALSE
+
+	if(target) // Check: Otherwise it runtimes if you fail to feed on someone.
+		if(amSilent)
+			to_chat(user, "<span class='notice'>You slowly release [target]'s wrist." + (target.stat == 0 ? " [target.p_their(TRUE)] face lacks expression, like you've already been forgotten.</span>" : ""))
+		else
+			user.visible_message("<span class='warning'>[user] unclenches their teeth from [target]'s neck.</span>", \
+								 "<span class='warning'>You retract your fangs and release [target] from your bite.</span>")
+		log_combat(owner, target, "fed on blood", addition="(and took [amount_taken] blood)")
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = user.mind.has_antag_datum(/datum/antagonist/bloodsucker)
 	// No longer Feeding
 	if(bloodsuckerdatum)
 		bloodsuckerdatum.poweron_feed = FALSE
+	// Reset ALL checks for next time the Power is used.
+	amSilent = FALSE
+	warning_target_inhuman = FALSE
+	warning_target_dead = FALSE
+	warning_full = FALSE
 	feed_target = null
 	// My mouth is no longer full
 	REMOVE_TRAIT(owner, TRAIT_MUTE, BLOODSUCKER_TRAIT)
```

Comment:
starting and then immediately ending a feeding session while in a frenzy will let you speak again without leaving your frenzy.

you'll still be deaf, though.